README.md

```diff
@@ -253,6 +253,7 @@
 - 🧩 [Sail UI](https://sailui.github.io/) - Collection of basic UI components built on Tailwind CSS.
 - 🧩 [jQuery Toggler](https://craigerskine.github.io/jquery-tailwind-checkbox-toggle) - Switches using jQuery and Tailwind CSS.
 - 🧩 [Tailwind Kit](https://creative-tim.com/learning-lab/tailwind-starter-kit) - Framework-agnostic, Vue.js, React and Angular components.
+- 🧩 [Free Tailwind CSS Templates](https://elegantgoodies.gumroad.com/l/tailwind-bundle) - 5 free well-ceded and elegant styled Tailwind CSS Templates
```

Comment:
Using the wrong emoji and needs to be at the end of the list.

Reply:
Hey!

How do I fix it?

Reply:
By updating your branch with the changes.